src/pages/_app.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { AppProps } from 'next/app';
+import Layout from '../layout';
+import Head from 'next/head';
+
+const TaoTalkApp = ({ Component, pageProps }: AppProps) => {
+  return (
+    <Layout>
+      <Head>
+        <title>TaoTalk</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="TaoTalk" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Component {...pageProps} />
+    </Layout>
```

Comment:
The Head component should not be inside of the Layout component, even the output code(after compile) looks okay.

Reply:
The Nextjs docs says: The contents of head get cleared upon unmounting the component, so make sure each page completely defines what it needs in head, without making assumptions about what other pages added.
Does that mean for every page we need to define Head component separately, even if there all habe the same Head?

Reply:
This Head component used on the _app.js file, which means all pages would implement it as a global component.

However, if you would like to define a different header, you need to overwrite it with another Head on the page you need.

eg. 
You can add 
<Head> <title> TaoTalk - blog </title> </Head>
on the blog page.

Reply:
Since different pages may have different head content (which is the normal scenario), like the "description" one, it cannot be the same just "TaoTalk" on other pages. And since each HTML file can only contain one head section, I think it is not necessary to define it in "_app.tsx" file.

Reply:
@tatoMa Got it. I followed the suggestion from @Prom1s1ngYoung to put the Head element in the page file. Other pages just need to update the information needed, such as the Title element.